pkg/kubelet/container/runtime.go

```diff
@@ -381,14 +381,23 @@ type PortMapping struct {
 	HostIP string
 }
 
+type DeviceInfo struct {
+	// Path on host for mapping
+	PathOnHost string
+	// Path in Container to map
+	PathInContainer string
+	// Cgroup permissions
+	Permissions string
+}
+
 // RunContainerOptions specify the options which are necessary for running containers
 type RunContainerOptions struct {
 	// The environment variables list.
 	Envs []EnvVar
 	// The mounts for the containers.
 	Mounts []Mount
 	// The host devices mapped into the containers.
-	Devices []string
+	Devices []DeviceInfo
```

Comment:
@Hui-Zhi how to populate `DeviceInfo` for GPU devices? does api.PodSpec needing changes for this?